process.c

```diff
@@ -3020,77 +3005,67 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
 
 /*
  *  call-seq:
- *     exec([env,] command... [,options])
- *
- *  Replaces the current process by running the given external _command_, which
- *  can take one of the following forms:
+ *    exec([env, ] command_line, options = {})
+ *    exec([env, ] exe_path, options  = {})
+ *    exec([env, ] exe_path, *args, options  = {})
```

Comment:
I think it's more accurate to say that `options` is keyword arguments.

```suggestion
 *    exec([env, ] command_line, **options)
 *    exec([env, ] exe_path, **options)
 *    exec([env, ] exe_path, *args, **options)
```

Reply:
As it can contain `IO` instances as keys, it isn’t strictly “keyword” arguments.

Reply:
I see. Yeah I think we can leave as-is.